package.json

```diff
@@ -34,6 +34,7 @@
   "devDependencies": {
     "babel-cli": "^6.10.1",
     "babel-eslint": "^6.0.5",
+    "babel-plugin-transform-regenerator": "^6.21.0",
```

Comment:
Removing causes build to fail. Appears to be dependency of `babel-preset-es2015`. Keeping it in.